src/MezzioInstaller/config.php

```diff
@@ -8,9 +8,6 @@
         'chubbyphp/chubbyphp-laminas-config' => [
             'version' => '^1.0',
         ],
-        'elie29/zend-phpdi-config'           => [
-            'version' => '^6.0',
```

Comment:
are we sure bridge is no longer supported? I can see latest release this year:

https://github.com/elie29/zend-di-config/releases

also - maybe the bug was here - as we use version 6, but we support already version 8:

https://github.com/mezzio/mezzio-skeleton/blob/ebf6808206ac299ccd532947e49da77bdae1120f/composer.json#L56

Reply:
probably, @elie29 in case you can make it working?

Reply:
@samsonasik 
The version must be updated: `'version' => '^6.0 || ^8.0'`
This should work because a `composer.json` is created from this configuration.

https://github.com/mezzio/mezzio-skeleton/blob/ebf6808206ac299ccd532947e49da77bdae1120f/src/MezzioInstaller/OptionalPackages.php#L484

https://github.com/mezzio/mezzio-skeleton/blob/ebf6808206ac299ccd532947e49da77bdae1120f/src/MezzioInstaller/OptionalPackages.php#L398-L414

Reply:
@samsonasik 
I think you missed my hint here.